src/constants.ts

```diff
@@ -220,3 +220,5 @@ To see this in action:
 4. Next, type "short" in the example autocomplete input field.
   - Notice how the user is presented with only short sleeved items as results.
   - This is because we are filtering to the "Shirts" group`;
+
+export const termsWithImagesAndCountsDescription = `Pass boolean flags for \`displaySearchSuggestionImages\` and \`displaySearchSuggestionResultCounts\` fields to display images and counts for search suggestions.`;
```

Comment:
Should we add a note here about making the fields displayable? So they won't just pass the flags and wonder why it doesn't work.

Reply:
That's a great idea. I'll update